db/fixtures/600_themes.rb

```diff
@@ -28,8 +28,8 @@
       )
   end
 
-  Theme.foundation_theme.set_default!
+  Theme.horizon_theme.set_default!
 
-  dark_scheme_id = ColorScheme.where(base_scheme_id: ColorScheme::NAMES_TO_ID_MAP["Dark"]).pick(:id)
-  Theme.foundation_theme.update!(dark_color_scheme_id: dark_scheme_id) if dark_scheme_id.present?
+  dark_scheme_id = ColorScheme.where(name: "Horizon Dark").pick(:id)
```

Comment:
@renato I do think this is the only way, but you could make it a bit more specific by doing this:

```suggestion
dark_scheme_id =  Theme.horizon_theme.color_schemes.find_by(name: "Horizon Dark")&.id
```